src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -4942,7 +4942,8 @@ private static bool CanReuseVariableDeclarator(CSharp.Syntax.VariableDeclaratorS
                             var expression = item as ExpressionSyntax;
                             if (expression != null)
                             {
-                                args.Add(_syntaxFactory.Argument(null, default(SyntaxToken), expression));
+                                args.Add(_syntaxFactory.Argument(null, default(SyntaxToken), expression,
+                                                                 type: null, identifier: default(SyntaxToken)));
```

Comment:
This is a different factory, it is for internal syntax nodes, the on in SyntaxFactory is for public nodes, I believe.  

---
In reply to: [63744398](https://github.com/dotnet/roslyn/pull/11393#discussion_r63744398) [](ancestors = 63744398)